spark/src/test/scala/org/apache/comet/exec/CometExecSuite.scala

```diff
@@ -1430,6 +1430,41 @@ class CometExecSuite extends CometTestBase {
       }
     })
   }
+
+  test("Windows support") {
+    Seq("true", "false").foreach(aqeEnabled =>
+      withSQLConf(
+        CometConf.COMET_EXEC_SHUFFLE_ENABLED.key -> "true",
+        SQLConf.ADAPTIVE_EXECUTION_ENABLED.key -> aqeEnabled) {
+        withTable("t1") {
+          val numRows = 10
+          spark
+            .range(numRows)
+            .selectExpr("id AS a", s"$numRows - id AS b") // Todo: Test Nulls
+            .repartition(3) // Force repartition to test data will come to single partition
+            .write
+            .saveAsTable("t1")
```

Comment:
This can be simplified a little with `withParquetTable()`

Reply:
Changed to `withParquetTable()`. Thanks